app/components/navigation/Nav.tsx

```
@@ -28,10 +33,17 @@ const Nav = () => {
           >
             <MenuTrigger isOpen={isOpen} onClick={() => setIsOpen(!isOpen)} />
-            <PopupMenu isOpen={isOpen} setIsOpen={setIsOpen} />
           </div>
         </div>
       </div>
+      {/* Backdrop overlay to close menu on tap outside */}
+      {isOpen && (
+        <div
+          className="fixed inset-0 z-40"
+          onClick={() => setIsOpen(false)}
+        />
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)